wire-ios/Wire-iOS/Sources/Authentication/Helpers/ZMUserSession+MarketingConsent.swift

```diff
@@ -21,6 +21,7 @@ import WireSyncEngine
 
 extension ZMUserSession {
     func submitMarketingConsent(with marketingConsent: Bool) {
-        ZMUser.selfUser().setMarketingConsent(to: marketingConsent, in: self, completion: { _ in })
+        guard let selfUser = ZMUser.selfUser() else { return assertionFailure("ZMUser.selfUser() is nil") }
```

Comment:
Can one say `return assertionFailure(...)` and doesn't it return then without running the assertion?
For my eyes it's easier to read when this broken into more lines of code..

Reply:
I preferred this style, because it fits in one line, otherwise it's four lines:
```
guard let selfUser = ZMUser.selfUser() else {
    assertionFailure("ZMUser.selfUser() is nil")
    return
}
```

Are you sure you want it changed everywhere?

Reply:
Me personally I would say if it's only `return` do inside the line, if it's more like `return false` use more lines. But I know it's highly discussable...
We can leave it here like you did and talk about it with the team?

Reply:
Changed it now to multiple lines.